src/lib/components/DisclaimerModal.svelte

```diff
@@ -0,0 +1,86 @@
+<script lang="ts">
+	import { base } from "$app/paths";
+	import { page } from "$app/stores";
+	import {
+		PUBLIC_APP_DATA_SHARING,
+		PUBLIC_APP_DESCRIPTION,
+		PUBLIC_APP_NAME,
+	} from "$env/static/public";
+	import LogoHuggingFaceBorderless from "$lib/components/icons/LogoHuggingFaceBorderless.svelte";
+	import Modal from "$lib/components/Modal.svelte";
+	import type { LayoutData } from "../../routes/$types";
+	import Logo from "./icons/Logo.svelte";
+
+	export let settings: LayoutData["settings"];
+</script>
+
+<Modal>
+	<div
+		class="flex w-full flex-col items-center gap-6 bg-gradient-to-b from-primary-500/40 via-primary-500/10 to-primary-500/0 px-4 pb-10 pt-9 text-center"
+	>
+		<h2 class="flex items-center text-2xl font-semibold text-gray-800">
+			<Logo classNames="mr-1" />
+			{PUBLIC_APP_NAME}
+		</h2>
+
+		<p
+			class="px-2 text-lg font-semibold leading-snug text-gray-800 sm:px-8"
+			style="text-wrap: balance;"
+		>
+			{PUBLIC_APP_DESCRIPTION}
+		</p>
+
+		<p class="px-2 text-sm text-gray-800">
+			Disclaimer: AI is an area of active research with known problems such as biased generation and
+			misinformation. Do not use this application for high-stakes decisions or advice.
+		</p>
```

Comment:
you can remove this completely

Reply:
Would it make more sense to set `PUBLIC_APP_DATA_SHARING=0` in our prod config rather than removing the feature? That way the feature still exists for other people that want to set it

Reply:
I selected the wrong text, it was the one about conversations sharing.

Reply:
Yes, I meant that the text about data sharing can be disabled if `PUBLIC_APP_DATA_SHARING=0` in `.env.template` :eyes:

Reply:
Yes, I removed it. But you are right, maybe for chat-ui it actually makes sense.